src/pages/user/index.tsx

```diff
@@ -30,11 +32,23 @@ const User: React.FunctionComponent<
   const [account, setAccount] = React.useState<any>({})
   const {stripe_customer_id} = account
   const {viewer, authToken} = useViewer()
+  const [progress, setProgress] = React.useState<any>([])
 
   const {email: currentEmail, accounts, providers} = viewer || {}
   const {slug} = get(accounts, '[0]', {})
   const isConnectedToGithub = providers?.includes('github')
 
+  React.useEffect(() => {
+    const loadProgressForUser = async (user_id: number) => {
```

Comment:
I didn't paginate this, but the data is paginated and limited to 5 right now. We could make it "infinite scroll" or something like that if we wanted.